docs/HowTo/Get-Started/Connect-To-Testnet.md

```diff
@@ -0,0 +1,184 @@
+---
+description: How to choose and connect to a testnet
+---
+
+# Connect to a Testnet
+
+The PegaSys Teku client allows you to connect a beacon-chain node to a public testnet and run validators on these networks. There are different ETH2 testnets to choose from.
+
+!!! note
+
+    Due to the fast pace of client and testnet development, this guide recommends connecting to the [Schlesi Testnet](https://github.com/goerli/schlesi). However, networks can experience stability issues and are prone to regular resets. It's recommended to regularly check the network's and client's documentation for updates.
+
+**Prerequisites**:
+
+* [Install Teku](/HowTo/Get-Started/Install-Binaries) Ethereum 2.0 client or [Build Teku from Source](/HowTo/Get-Started/Build-From-Source).
```

Comment:
links have to point to the relative .md source files, not the URLs generated by MkDocs.
Here it should be `/HowTo/Get-Started/Install-Binaries.md` and `/HowTo/Get-Started/Build-From-Source.md`